### ISSUE TITLE
Increasing Performance

### ISSUE THREAD

**Issue by Edward334** [xuebinqin/U-2-Net#220]
Hi, 

First of all, thank you for the very inspired work on this network. It's wonderful to see how many applications it has and will surely have in the future. 

I am raising this issue to ask if you could provide some tips or advice on how to increase performance for a segmentation task. I will provide 3 examples below. They are all results of the same model. This model was trained for around 1.5 millions iterations and it had the lowest loss value from all the saved models. 

![image](https://user-images.githubusercontent.com/26660549/122536841-a1b48480-d02d-11eb-8909-9cac01b4d137.png)
![image](https://user-images.githubusercontent.com/26660549/122537105-e50ef300-d02d-11eb-879e-1c697c65c19f.png)
![image](https://user-images.githubusercontent.com/26660549/122536772-919ca500-d02d-11eb-831e-b2c9e0cd9d0d.png)

The first picture is close to perfect, as is a good chunk of the test set. 
However, there are examples on which the model is performing very poorly. While it is evident that the second and third pictures are loss obvious, and the subjects blend more with the background, it makes we wonder why the huge difference in performance.

In conclusion, can you provide any advice on how to improve the model and have it perform better for cases like this ? I tried database augmentation, and increasing the input resolution. I also tried experimenting with different cost functions such as L1 L2 and mIoU with no better results.
